sample-service/src/main/java/uk/ac/ox/ndph/mts/sample_service/security/authorisation/SecurityContextComponent.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ox.ndph.mts.sample_service.security.authorisation;
+
+import com.microsoft.azure.spring.autoconfigure.aad.UserPrincipal;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+/**
+ * Security context service to get data from the security context
+ */
+@Component
+public class SecurityContextComponent {
```

Comment:
Can you find a better name for this?
(also, the comment doesn't fit the name, and I don't think that you need "component" in its name)